t/plugin/limit-conn2.t

```diff
@@ -410,32 +440,102 @@ GET /t
 
 
 
-=== TEST 11: bypass empty key
+=== TEST 11: request when key is missing
+--- request
+GET /test_concurrency
+--- timeout: 10s
+--- response_body
+200
+503
+503
+503
+503
+--- no_error_log
+[error]
+--- error_log
+The key of limit conn is empty or invalid, set $remote_addr as the key
+
+
+
+=== TEST 12: update plugin to set invalid key
 --- config
     location /t {
         content_by_lua_block {
-            local json = require "t.toolkit.json"
-            local http = require "resty.http"
-            local uri = "http://127.0.0.1:" .. ngx.var.server_port
-                        .. "/limit_conn"
-            local ress = {}
-            for i = 1, 2 do
-                local httpc = http.new()
-                local res, err = httpc:request_uri(uri)
-                if not res then
-                    ngx.say(err)
-                    return
-                end
-                table.insert(ress, res.status)
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "limit-conn": {
+                                "conn": 1,
+                                "burst": 0,
+                                "default_conn_delay": 0.1,
+                                "rejected_code": 503,
+                                "key": "abcdefgh",
+                                "key_type": "var_combination"
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1980": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/limit_conn"
+                }]],
+                [[{
```

Comment:
There is no need to check the response, right?

Reply:
Sorry, I do not understand this point? Could you give me more infos?

Reply:
Do you mean that there is no need to check this?
```
--- response_body
passed
```